package.json

```diff
@@ -33,6 +33,7 @@
     "@strapi/utils": "^5.8.0",
     "axios": "^1.8.4",
     "generate-password": "^1.7.1",
+    "oauth4webapi": "^3.3.2",
```

Comment:
I propose using [oauth4webapi](https://github.com/panva/oauth4webapi) instead of pkce-challenge.
oauth4webapi is a [Relying Party library certified for OpenID Connect](https://openid.net/developers/certified-openid-connect-implementations/).